python/ml4ir/applications/classification/tests/data/configs/entity_id_vocab.csv

```diff
@@ -0,0 +1,10 @@
+key
+""
```

Comment:
Why do we need this value?

Reply:
Up to you really. This allows you to differentiate between an empty entity_id in the user context versus an out of vocab entity_id.